src/Castle.Windsor.Extensions.MsDependencyInjection/NetCoreScopeAccessor.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2004-2020 Castle Project - http://www.castleproject.org/
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Castle.Windsor.Extensions.MsDependencyInjection
+{
+    using System;
+    
+    using Castle.MicroKernel.Context;
+    using Castle.MicroKernel.Lifestyle.Scoped;
+
+    internal class NetCoreScopeAccessor : IScopeAccessor
+    {
+        public ILifetimeScope GetScope(CreationContext context)
+        {
+            if(NetCoreScope.Current == null)
+            {
+                throw new InvalidOperationException("No scope available");
+            }
+            return NetCoreScope.Current;
```

Comment:
According to the coding style, ` ??` is not allowed in Castle project

Reply:
The coding style document predates the `??` operator, it doesn't even mention it. We haven't had the coding style document even mentioned in the last few years other than for tabs rather than spaces. What do we need to remove from the style guide to make this clearer?